substrate/cmd/kitctl/bootstrap.go

```diff
@@ -51,19 +51,19 @@ func bootstrap(cmd *cobra.Command, args []string) {
 	start := time.Now()
 	name := parseName(ctx, args)
 	logging.FromContext(ctx).Infof("Bootstrapping %q", name)
-	vpcCidrs := []string{"10.0.0.0/16", "10.1.0.0/16", "10.2.0.0/16", "10.3.0.0/16", "10.4.0.0/16"}
 	if err := substrate.NewController(ctx).Reconcile(ctx, &v1alpha1.Substrate{
 		ObjectMeta: metav1.ObjectMeta{Name: name},
 
 		Spec: v1alpha1.SubstrateSpec{
-			VPC:          &v1alpha1.VPCSpec{CIDR: vpcCidrs},
+			VPC:          &v1alpha1.VPCSpec{CIDR: []string{"10.0.0.0/16"}},
```

Comment:
can you document saying the KIT Guest clusters that are launched cannot exceed the IPs more than 64K combined ( worker nodes + pods) ? Basically cx should be aware of this limitation if we take this route.